code/_globalvars/regexes.dm

```diff
@@ -0,0 +1,7 @@
+//These are a bunch of regex datums for use /((any|every|no|some|head|foot)where(wolf)?\sand\s)+(\.[\.\s]+\s?where\?)?/i
+GLOBAL_DATUM_INIT(is_http_protocol, /regex, regex("^https?://"))
```

Comment:
is_(h)yper(t)ext(t)ransfer(p)rotocol_protocol

Reply:
I'll correct this after I make a quick stop by the ATM machine, I may be a while though because I forgot my PIN number.